src/App.tsx

```diff
@@ -1,61 +1,183 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
 
 export const App = () => {
+  const [todoTitle, setTodoTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+
+  const [hasUserError, setHasUserError] = useState(false);
+  const [hasTodoTitleError, setHasTodoTitleError] = useState(false);
+
+  const userNotFound = {
+    id: 0,
+    name: 'user',
+    email: 'user@mail.com',
+  };
```

Comment:
In case if user is not found - set null for it, no need to mock objects
What if there will be many not found users (with 0 id)?